app/questionnaire/placeholder_parser.py

```diff
@@ -81,11 +81,19 @@ def __call__(
         self, placeholder_list: Sequence[Mapping]
     ) -> MutableMapping[str, ValueSourceEscapedTypes | ValueSourceTypes]:
         placeholder_list = QuestionnaireSchema.get_mutable_deepcopy(placeholder_list)
```

Comment:
@Yuyuutsu why mutable deepcopy? it doesn't look like your placeholder list is ever modified after?

Reply:
Good question. Ill have a look into this

Reply:
Had a look into it, not sure why it was here change was three years ago. Removed and working as expected 👍